### PR TITLE
Refactors

### PR DESCRIPTION
**Pull request by cmc333333**
Two small refactors to remove unnecessary code. See changesets for specifics

### CODE REVIEW

**Comment by cmc333333** [comment]
Coverage dropped due to removing unused functionality. I'll squeeze in a new test file to make up for it, though...

**Comment by cmc333333** [comment]
Alllllright. These little refactors got a bit larger as I tried to keep coverage relatively high. Individual changesets are still probably the best way to step through these changes.

File: regcore/tests/db_django_models_tests.py [hunk 1]
```diff
@@ -7,7 +7,11 @@
 from regcore.models import Diff, Layer, Notice, Regulation
 
 
-class ReusableDMRegulations(object):
+class DMRegulationsTest(TestCase):
+    def setUp(self):
+        Regulation.objects.all().delete()
```
Comment (jmcarp) [inline regcore/tests/db_django_models_tests.py]: Django `TestCase` tests run within a transaction, right? Do we need these `delete` queries? The tests seem to pass without them.

Comment (cmc333333) [inline regcore/tests/db_django_models_tests.py]: Good point

**Comment by jmcarp** [comment]
screen shot 2016-03-04 at 11 35 49 am

:tada:

**Pull request merged**